bigquery/google/cloud/bigquery/magics.py

```diff
@@ -274,6 +289,16 @@ def _run_query(client, query, job_config=None):
         "Standard SQL if this argument is not used."
     ),
 )
+@magic_arguments.argument(
+    "--use_bqstorage_api",
+    action="store_true",
+    default=False,
+    help=(
+        "[Beta] Use the BigQuery Storage API to download large query results. "
+        "To use this option, install the BigQuery Storage client library, "
```

Comment:
Update to clear and actionable phrasing: "install the google-cloud-bigquery-storage and fastavro libraries"
Also, add this to the docstring at the top of the file.

Reply:
Updated docstrings in https://github.com/googleapis/google-cloud-python/pull/7640/commits/56bcb0569ce24154fcd830a3ec7c828d3198a2bf